src/pages/portfolio/index.tsx

```diff
@@ -1,11 +1,46 @@
 'use client';
 
+import React from 'react';
+import { XCircle } from 'lucide-react';
+import { Button } from '@penumbra-zone/ui/Button';
 import { Text } from '@penumbra-zone/ui/Text';
+import { Density } from '@penumbra-zone/ui/Density';
 
-export const PortfolioPage = () => {
-  return (
-    <section>
-      <Text h2>Hi!</Text>
+export const PortfolioPage = (props: { isMobile: boolean }) => {
+  return props.isMobile ? (
```

Comment:
issue: instead of a large ternary can you make two components? Maybe `<MobilePortfolioPage />` & `<DesktopPortfolioPage />`?

Reply:
done